tests/blockchain/eth/contracts/main/miners_escrow/conftest.py

```diff
@@ -21,18 +21,14 @@
 
 
 VALUE_FIELD = 0
-DECIMALS_FIELD = 1
-CONFIRMED_PERIOD_1_FIELD = 2
-CONFIRMED_PERIOD_2_FIELD = 3
-LAST_ACTIVE_PERIOD_FIELD = 4
 
 secret = (123456).to_bytes(32, byteorder='big')
 
 
 @pytest.fixture()
 def token(testerchain):
     # Create an ERC20 token
-    token, _ = testerchain.interface.deploy_contract('NuCypherToken', 2 * 10 ** 9)
+    token, _ = testerchain.interface.deploy_contract('NuCypherToken', _initialAmount=2 * 10 ** 9)
```

Comment:
In places like this, it can be useful to use the `NU` class:
```suggestion
    token, _ = testerchain.interface.deploy_contract('NuCypherToken', _initialAmount=int(NU(2 * 10 ** 9, 'NuNit')))
```
By the way, that amount is very low, it's not even close to 1 NU. Shouldn't we increment it to certain amount of NU?

```suggestion
    token, _ = testerchain.interface.deploy_contract('NuCypherToken', _initialAmount=int(NU(2_000_000, 'NU')))
```

Reply:
Some tests and coefficient use this value so changing it leads to test fails. So I used first suggestion

Reply:
Why that amount though? It's not in the right order of magnitude of a real token distribution. Also, if it's used in other tests, perhaps it makes sense to use a constant.

Reply:
It was chosen together with mining coefficient to have small reward in tests and also different for some test cases. For contracts it's does not matter how many zeros (except tests economics). I think it will be more clear to use `TokenEconomics` for that constants (#936)